core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -498,15 +497,4 @@ default void addBulkLoadInProgressFlag(String path, long fateTxid) {
   default void removeBulkLoadInProgressFlag(String path) {
     throw new UnsupportedOperationException();
   }
-
-  /**
-   * Remove all the Bulk Load transaction ids from a given table's metadata
-   *
-   * @param tableId Table ID for transaction removals
-   * @param tid Transaction ID to remove
-   */
-  default void removeBulkLoadEntries(TableId tableId, long tid) {
```

Comment:
This is going to conflict with #3336 . You might want to wait to merge this until #3336 is merged into 2.1 and up to elasticity.